EasyCommands/BlockHandlers/TextSurfaceHandlers.cs

```diff
@@ -63,11 +63,11 @@ public class TextSurfaceHandler : MultiInstanceBlockHandler<IMyTextSurface> {
             public override string Name(IMyTextSurface block) => block.DisplayName;
 
             public override IEnumerable<IMyTextSurface> GetInstances(IMyTerminalBlock block) =>
-                block is IMyTextSurface ?
-                Enumerable.Repeat((IMyTextSurface)block, 1) :
-                block is IMyTextSurfaceProvider ?
-                Enumerable.Range(0, ((IMyTextSurfaceProvider)block).SurfaceCount).Select(i => ((IMyTextSurfaceProvider)block).GetSurface(i)) :
-                Enumerable.Empty<IMyTextSurface>();
+                block is IMyTextSurface
+                    ? Once((IMyTextSurface)block)
```

Comment:
nit: I think you could use NewList() here to save a few chars

Reply:
Rather, why is the Once method necessary, given ```NewList(item)``` ?

Reply:
no allocation that gets thrown away immediatly since that expression is expanded in a `SelectMany` anyway.

Reply:
also the branches inside `?:` must have the same type